deploy/agent-vm.mk

```diff
@@ -0,0 +1,28 @@
+bin/output/disk.qcow2: hack/Containerfile.local
+	sudo podman build -f hack/Containerfile.local -t localhost/local-flightctl-agent:latest .
+	mkdir -p bin/output
+	sudo podman run --rm -it --privileged --pull=newer --security-opt label=type:unconfined_t \
+	                -v $(shell pwd)/bin/output:/output -v /var/lib/containers/storage:/var/lib/containers/storage \
+					 quay.io/centos-bootc/bootc-image-builder:latest --type qcow2 \
+					 --local localhost/local-flightctl-agent:latest
+
+agent-image: bin/output/disk.qcow2
+	@echo "Agent image built at bin/output/disk.qcow2"
+
+.PHONY: agent-image
+
+agent-vm: bin/output/qcow2/disk.qcow2
+	@echo "Booting Agent VM from bin/output/qcow2/disk.qcow2"
+	sudo cp bin/output/qcow2/disk.qcow2 /var/lib/libvirt/images/disk.qcow2
+	sudo chown qemu:qemu /var/lib/libvirt/images/disk.qcow2
+	sudo virt-install --name flightctl-device --vcpus 2 --memory 1024     --import --disk /var/lib/libvirt/images/disk.qcow2,format=qcow2 \
```

Comment:
minor: can we put each flag on a newline  please :)

Reply:
minor: Makefile take a param for VIRT_DISK or similar?

Reply:
hmmm we can accept a VMNAME to let start multiple instances